<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QWidget" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>开始时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateStartTime">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>结束时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateOverTime">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushQuery">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClear">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClearHistory">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>删除历史记录</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabSum">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabAir">
      <attribute name="title">
       <string>空气温湿度</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabLight">
      <attribute name="title">
       <string>光照强度</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabSoil">
      <attribute name="title">
       <string>土壤湿度</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
